Software/real_time/ROS_RoboBuggy/src/robobuggy/src/transistor/gps/GPS_Broadcaster_Runner.cpp

```diff
@@ -10,7 +10,13 @@ int main(int argc, char **argv)
     ros::NodeHandle nh;
 
     GPS_Broadcaster broadcaster;
-    int err = broadcaster.handle_serial_messages();
-
-    return err;
+    while (ros::ok()) {
```

Comment:
Might want to introduce a consistent update rate here, so that we don't waste resources spinning